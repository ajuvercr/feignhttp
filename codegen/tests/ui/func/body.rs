use feignhttp_codegen::post;

// error: request must have only one body
//    |  pub async fn send_post(#[body] b: B, #[body] b2: B) -> Result<String> {}
//    |                                 ^^^^^^^^^^^^^^^^^^^

struct B;

#[post("http://xxx")]
pub async fn send_post(#[body] b: B, #[body] b2: B) -> Result<String> {}

fn main() {}
