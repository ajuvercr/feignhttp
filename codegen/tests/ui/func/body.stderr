error: request must have only one body
  --> tests/ui/func/body.rs:10:32
   |
10 | pub async fn send_post(#[body] b: B, #[body] b2: B) -> Result<String> {}
   |                                ^^^^^^^^^^^^^^^^^^^
